src/main/java/com/knu/buga1chuk/serialization/service/BinaryPersonSerializationService.java

```diff
@@ -45,4 +48,8 @@ public List<Person> getPersonFromFile(File file) {
         return personList.getPersons();
     }
 
+    @Override
+    public Person toPerson(String input) {
+        throw new UnsupportedOperationException("Conversion is not supported");
```

Comment:
Check the `base64 encoding` topic.
It is common way to show the binary data to human.
Implement this method with help of the base64 encoding.

Reply:
fix